front/src/routes/integration/all/withings/device-page/actions.js

```diff
@@ -18,7 +18,7 @@ const actions = store => ({
     const deviceSaved = await state.httpClient.post('/api/v1/device', device);
     device.inDB = true;
     device.selector = deviceSaved.selector;
-    state.httpClient.get(`/api/v1/service/withings/poll/${device.selector}`);
+    state.httpClient.get(`/api/v1/service/withings/post_create/${device.selector}`);
```

Comment:
That's not what I meant.
In Gladys core, once you crete or update a device, method `postCreate` or `postUpdate` is called.
So you don't need a WS call to run something after its creation.
You just need to implement `postCreate` or `postUpdate` method in your service at backend side.

Reply:
ok I put a postCreate function